lms/djangoapps/discussion_api/tests/test_serializers.py

```diff
@@ -586,6 +587,7 @@ def test_update_all(self):
                 "closed": ["False"],
                 "pinned": ["False"],
                 "user_id": [str(self.user.id)],
+                "read": ["False"],
```

Comment:
Which unit test tries to update the `read` field from `False` to `True`?  Perhaps add the "read" field to the `data` dict on L570 and make sure it gets set?